packages/server/src/server.ts

```diff
@@ -1,3 +1,4 @@
+import "dotenv/config";
```

Comment:
why do we want to add dotenv here? this is more a package that we can add to the application code